### ISSUE TITLE
I could not generate the music app not responding

### ISSUE THREAD

**Issue by srirammurty** [ace-step/ACE-Step#316]
I have installed the app as per the installations guidelines. I could able to launch the Gradio interface. When I tried to generate for first time it downloaded all the models required from hugging face. However the app keep running infinitely but not generating any music. I am using Windows 11 with Python 3.11.  

**(venv) C:\Sriram\Multi\Music_generation\Application\ACE-Step>acestep --port 7865
* Running on local URL:  http://127.0.0.1:7865
* To create a public link, set `share=True` in `launch()`.
2025-08-20 22:42:26.629 | WARNING  | acestep.pipeline_ace_step:__call__:1480 - Checkpoint not loaded, loading checkpoint...
2025-08-20 22:42:26.630 | INFO     | acestep.pipeline_ace_step:get_checkpoint_path:178 - Download models from Hugging Face: ACE-Step/ACE-Step-v1-3.5B, cache to: C:\Users\Sriram\.cache/ace-step/checkpoints
Fetching 14 files: 100%|███████████████████████████████████████████████████████████████████████| 14/14 [00:00<?, ?it/s]
C:\Sriram\Multi\Music_generation\Application\ACE-Step\venv\Lib\site-packages\torch\nn\utils\weight_norm.py:144: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
2025-08-20 22:42:56.592 | INFO     | acestep.pipeline_ace_step:__call__:1488 - Model loaded in 29.96 seconds.
2025-08-20 22:42:57.331 | INFO     | acestep.pipeline_ace_step:__call__:1536 - random audio duration: 162.43691869438297
2025-08-20 22:42:57.332 | INFO     | acestep.pipeline_ace_step:text2music_diffusion_process:847 - cfg_type: apg, guidance_scale: 15, omega_scale: 10
2025-08-20 22:42:57.359 | INFO     | acestep.pipeline_ace_step:text2music_diffusion_process:1072 - start_idx: 15, end_idx: 45, num_inference_steps: 60
  0%|                                                                                           | 0/60 [00:00<?, ?it/s]**

After this no more update  . Please help to fix this

![Image](https://github.com/user-attachments/assets/8bba2098-8ad7-48dc-98da-9cd505c0f4a8)
